@xen-orchestra/lite/src/components/vm/VmHeader.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <TitleBar :icon="faDisplay">
+    {{ name }}
+    <template #actions>
+      <AppMenu shadow placement="bottom-end">
+        <template #trigger="{ open, isOpen }">
+          <UiButton :active="isOpen" :icon="faPowerOff" @click="open">
+            {{ $t("change-state") }}
+            <UiIcon :icon="faAngleDown" />
+          </UiButton>
+        </template>
+        <MenuItem :disabled="!isHalted" :icon="faPlay">
+          {{ $t("start") }}
+        </MenuItem>
+        <MenuItem :disabled="!isHalted" :icon="faServer">
+          {{ $t("start-on-host") }}
+          <template #submenu>
+            <MenuItem
+              v-for="host in hostStore.allRecords"
+              v-bind:key="host.$ref"
+              :icon="faServer"
+            >
+              <div class="wrapper">
+                {{ host.name_label }}
+                <div>
+                  <UiIcon
+                    :icon="
+                      host.$ref === poolStore.pool?.master ? faStar : undefined
+                    "
+                    class="star"
+                  />
+                  <PowerStateIcon
+                    :state="isHostRunning(host) ? 'Running' : 'Halted'"
+                  />
+                </div>
+              </div>
+            </MenuItem>
+          </template>
+        </MenuItem>
+        <MenuItem :disabled="!isRunning" :icon="faPause">
+          {{ $t("pause") }}
+        </MenuItem>
+        <MenuItem :disabled="!isRunning" :icon="faMoon">
+          {{ $t("suspend") }}
+        </MenuItem>
```

Comment:
Can we suspend a paused VM? If so, do we want to expose it in the UI? /cc @olivierlambert

Reply:
no, suspend is only for a running VM. Yes, we can expose it.